### DISCUSSION TITLE
You exceeded your current quota

### DISCUSSION THREAD

**Discussion by icougil** [waylaidwanderer/node-chatgpt-api#168] [Discussions]
Hi, 
I was trying to use your app, but I always receive the following error:

```
    Error: {"message":"You exceeded your current quota, please check your plan and billing details.","type":"insufficient_quota","param":null,"code":null}
```

I am wondering, is there anything not working as expected?

**Comment by waylaidwanderer** [comment]
Check your current usage: https://platform.openai.com/account/usage

OpenAI wasn't billing people for gpt-3.5-turbo usage before, and it seems like they've updated it so that your usage is counted now. So if you went over the monthly limit or have no more free credits, you'll run into that error.

**Reply by icougil** [reply]
Ah, got it.
Then,.. if I understand it correctly, I still have some credits, right?
![image](https://user-images.githubusercontent.com/2370634/223396156-1da51695-ba2c-43a0-bb5b-c245cc26d112.png)

**Reply by waylaidwanderer** [reply]
It says right in your screenshot that those credits expired on March 1st.

**Reply by icougil** [reply]
oh gosh!
... but meanwhile I can still use it via web?

**Reply by waylaidwanderer** [reply]
Web where? Playground or chat.openai.com? The latter is a separate service for consumers so it's billed separately.

**Reply by icougil** [reply]
The latter... ook... so basically you can use the web as you wish... but not the API ... 🤔
